packages/toolkit/src/query/core/buildSelectors.ts

```diff
@@ -1,4 +1,5 @@
-import { createNextState, createSelector } from './rtkImports'
+import type { createSelector as _createSelector } from './rtkImports'
```

Comment:
```suggestion
import { createSelector as _createSelector } from './rtkImports'
```

Reply:
Oh, nvm. TIL - you can use `import type` for objects you intended to use only for typing purposes (e.g. `typeof _createSelector`). Cool~